org.civicrm.reqangular/src/common/modules/angularDate.js

```diff
@@ -0,0 +1,8 @@
+define([
+    'common/angular',
+    'common/modules/settings'
```

Comment:
Why coupling this module to the settings? Only the actual app should be concerned with the settings, what's the benefit of creating a direct dependency here?